suzieq/db/parquet/parquetdb.py

```diff
@@ -620,48 +641,43 @@ def _get_filtered_fileset(self, datasets: list, namespace: list) -> ds:
         Returns:
             ds: pyarrow dataset of only the files that match filter
         """
-        filelist = []
-        for dataset in datasets:
-            if not namespace:
-                filelist.extend(dataset.files)
-                continue
-
-            notlist = [x for x in namespace
-                       if x.startswith('!') or x.startswith('~!')]
-            if notlist != namespace:
-                newns = [x for x in namespace
-                         if not x.startswith('!') and not x.startswith('~!')]
-            else:
-                newns = namespace
-            ns_filelist = []
-            chklist = dataset.files
-            for ns in newns or []:
+        if not namespace:
+            return dataset
+
+        notlist = [x for x in namespace
```

Comment:
Renaming this variable to `excluded_ns` probably allows us to make code more readeable
```suggestion
        excluded_ns = [x for x in namespace
```

Reply:
Ack